### PR TITLE
[py-wasabi] added version 1.1.2

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

**Comment by qwertos** [comment]
@spackbot fix style

File: var/spack/repos/builtin/packages/py-wasabi/package.py [hunk 1]
```diff
@@ -13,6 +13,10 @@ class PyWasabi(PythonPackage):
     homepage = "https://ines.io/"
```
Comment (manuelakuhn) [inline var/spack/repos/builtin/packages/py-wasabi/package.py]: According to pypi the homepage is
```suggestion
    homepage = "https://github.com/explosion/wasabi"
```

File: var/spack/repos/builtin/packages/py-wasabi/package.py [hunk 2]
```diff
@@ -13,6 +13,10 @@ class PyWasabi(PythonPackage):
...
     pypi = "wasabi/wasabi-0.6.0.tar.gz"
 
+    version("1.1.2", sha256="1aaef3aceaa32edb9c91330d29d3936c0c39fdb965743549c173cb54b16c30b5")
     version("0.6.0", sha256="b8dd3e963cd693fde1eb6bfbecf51790171aa3534fa299faf35cf269f2fd6063")
 
     depends_on("py-setuptools", type="build")
```
Comment (manuelakuhn) [inline var/spack/repos/builtin/packages/py-wasabi/package.py:19]: Can you also add
```
colorama >= 0.4.6; sys_platform == "win32" and python_version >= "3.7"
```
please?

File: var/spack/repos/builtin/packages/py-wasabi/package.py (hunk 1 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-wasabi/package.py]: Optional. Always good to update, but shouldn't hold up the PR from being merged.

**Comment by qwertos** [comment]
@spackbot fix style

**Comment by qwertos** [comment]
Depends on: #38737

**Comment by manuelakuhn** [comment]
If an additional package needs to be updated for a PR to work (like here with `py-colorama`) you don't have to open a separate PR for it. You can just add it here :-)  The policy regarding that changed a while back.

**Review by manuelakuhn — DISMISSED** [review]
LGTM and build on Fedora37 with gcc@12.3.1

File: var/spack/repos/builtin/packages/py-wasabi/package.py [hunk 3]
```diff
@@ -10,9 +10,16 @@
...
+    depends_on(
+        "py-typing-extensions@3.7.4.1:4.4", type=("build", "run"), when="@1.1.2:^python@:3.7"
+    )
+    depends_on(
+        "py-colorama@0.4.6:", type=("build", "run"), when="@1.1.2:^python@3.7:platform=windows"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-wasabi/package.py]:
```suggestion
        "py-colorama@0.4.6:", type=("build", "run"), when="@1.1.2: platform=windows"
```
Spack only supports installing 3.7+ so it's basically always required

**Review by manuelakuhn — DISMISSED** [review]

**Comment by qwertos** [comment]
@spackbot fix style

**Comment by qwertos** [comment]
@spackbot fix style

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**